dojo/tools/trivy/parser.py

```diff
@@ -126,6 +126,7 @@ def get_result_items(self, test, results, service_name=None):
                     vuln_id = vuln.get('VulnerabilityID', '0')
                     package_name = vuln['PkgName']
                     severity = TRIVY_SEVERITIES[vuln['Severity']]
+                    file_path = vuln['PkgPath']
```

Comment:
```suggestion
                    file_path = vuln.get('PkgPath')
```

Reply:
Do I need to write a new test case? Or use the given tests?

Reply:
both options are good, just add new checks somewhere that test the new data